conversion/conversion.go

```diff
@@ -216,7 +217,14 @@ func dataFromSQL(driver string, config spanner.BatchWriterConfig, client *sp.Cli
 func schemaFromDynamoDB(sampleSize int64) (*internal.Conv, error) {
 	conv := internal.MakeConv()
 	mySession := session.Must(session.NewSession())
-	client := dydb.New(mySession)
+	cfg := aws.Config{}
+	endpointOverride := os.Getenv("DYNAMODB_ENDPOINT_OVERRIDE")
+	if endpointOverride != "" {
+		cfg = aws.Config{
+			Endpoint: aws.String(endpointOverride),
+		}
+	}
+	client := dydb.New(mySession, &cfg)
```

Comment:
consider moving this to a common function

Reply:
Done